admin/remoter.go

```diff
@@ -18,7 +18,7 @@ type RemoteOptions struct {
 type Remoter interface {
 	// Exec allows a user to execute a command (usually tunneling) into a running service
 	// This only makes sense for container-based providers
-	Exec(ctx context.Context, options RemoteOptions, cmd []string) error
+	Exec(ctx context.Context, options RemoteOptions, cmd []string, username string) error
```

Comment:
It might make more sense to put this in the `admin.RemoteOptions`

Reply:
Done